spec/support/certificate_helpers.rb

```diff
@@ -27,4 +27,16 @@ def encrypted_secret_key_password
   def invalid_cert
     OpenSSL::X509::Certificate.new(File.read('spec/support/certificates/too_short_cert.crt'))
   end
+
+  def begin_end_cert(cert)
+    "-----BEGIN CERTIFICATE-----\n" +
+      cert +
+    "\n-----END CERTIFICATE-----\n"
+  end
```

Comment:
(suggestion, non-blocking): Instead of this, can we use a Heredoc?

```ruby
def add_cert_boundaries(cert_body)
    <<~TEXT
    -----BEGIN CERTIFICATE-----
    #{cert}
    -----END CERTIFICATE-----
    TEXT
end
```

Reply:
yup! updated in https://github.com/18F/saml_idp/pull/124/commits/8658a88a0e129e2a06f451b4e63e65c2dffa09ce